### PR TITLE
Celina

### PR DESCRIPTION
**Pull request by CelinaRiegel**
exercise_1.beendet

### CODE REVIEW

File: Exercises/CelinaRiegel/first_test.py
Comment (RubelOS) [inline Exercises/CelinaRiegel/first_test.py:1]: Should be removed

File: Exercises/CelinaRiegel/exercise_01.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# use this template for completing todays exercises
+# make a copy of the file, in a folder with your own name
+# add code below to answer the questions; for each question there should be
+# single print() which will give the answer(s) to that question.
+
+# what are the quotient and remainder after (integer) division of 2711274328912 by 23369245575
+print("quotient=", 2711274328912//23369245575)
+print("remainder=", 2711274328912%23369245575)
```
Comment (RubelOS) [inline Exercises/CelinaRiegel/exercise_01.py:8]: looks good! Keep an eye on the task "single print()"
you can use `print(solution1 \n solution2)`!!!

File: Exercises/CelinaRiegel/exercise_01.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+# which and how many elements are common to the two lists above? 
+gleich=0
+print("common elements:")
+for i in first_list:
+    for j in second_list:
+        if(i==j):
+            gleich+=1
+            print(i)
+print(gleich, "elements are the same")
```
Comment (RubelOS) [inline Exercises/CelinaRiegel/exercise_01.py:32]: keep in mind that u can use Pytohn's integrated functions. Your solution may be inefficient but still good!

File: Exercises/CelinaRiegel/exercise_01.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+# how often does the character 'a' occur in each of the lists above?
+letter="a"
+anzahl=0
+anzahl_zwei=0
+for i in first_list:
+    anzahl_zwei+=i.count("a")
+for i in second_list:
+    anzahl+=i.count("a")
```
Comment (RubelOS) [inline Exercises/CelinaRiegel/exercise_01.py:41]: same as before. Still good! love it!

**Review by RubelOS — CHANGES_REQUESTED** [review]
Pls remove the empty file.

The whole thing looks perfect! 👌👌🥰🥰🥰😶‍🌫️😶‍🌫️😶‍🌫️🫡🫡🫡🫡🤡👽

**Comment by RubelOS** [comment]
Why did u delete your exercise_01.py too?

File: Exercises/Celina/exercise_01.py [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+# which and how many elements are common to the two lists above? 
+gleich=0
+print("common elements:")
+for i in first_list:
+    for j in second_list:
+        if(i==j):
+            gleich+=1
+            print(i)
+print(gleich, "elements are the same")
```
Comment (RubelOS) [inline Exercises/Celina/exercise_01.py:32]: next time you can use inbuilt functions like len() of the intersect of both lists. For being able to use intersect() u have to convert both lists to a set

**Review by RubelOS — APPROVED** [review]
good solution

**Comment by atrejut** [comment]
@CelinaRiegel feel free to merge this :)

**Pull request merged**